### ISSUE TITLE
bounding box error and some questions about implementation

### ISSUE THREAD

**Issue by anonymre** [thtrieu/darkflow#568]
Hello thtrieu
First of all great work and thank you for making it available for everyone.
I am a newbie in neural networks trying to make darkflow recognise a number on a paper. i changed the labels.txt with digits 0-9 and a new cfg file with classes as 10 and filters as 75. after training with epoches 1125 still loss value doesn't get to zero (values stays around 1~3 even after making noise augmentation to none) . However i gave only one picture with all the digits in it with annotation values generated for only those digits . I tried with same image but still no bounding boxes. 
Doubts i have:
1. when we can train with own dataset does it mean it should be only person or dog with different pictures.
2. I tried to modify "test_darkflow"  python file with changing values and labels - did i do any mistake (can't we do anything like that?)
3. Does loss value should be around zero values and even after getting values around 1~3 they don't get saved (when i tried to train again the loss value starts again at 113?
4. Do we have to create new weight files if i try to make it recognize digits or other than any categories you have used?
5. If the size values are very less like the data i posted then can't we make it recognise?
6. I even tried to replace original annotation values with the one i generated  along with the image-did i do anything wrong?
I know these questions are silly but please help `
```
<annotation>
 <size>
<width>972</width>
		<height>4128</height>
		<depth>3</depth>
	</size>
	<segmented>0</segmented>
	<object>
		<name>9</name>
		<pose>Left</pose>
		<truncated>0</truncated>
		<difficult>0</difficult>
		<bndbox>
			<xmin>291</xmin>
			<ymin>805</ymin>
			<xmax>491</xmax>
			<ymax>934</ymax>
		</bndbox>
	</object>
</annotation>

```
